lib/Backend/IRBuilder.cpp

```diff
@@ -1698,35 +1732,6 @@ IRBuilder::BuildReg2(Js::OpCode newOpcode, uint32 offset, Js::RegSlot R0, Js::Re
 
     switch (newOpcode)
     {
-    case Js::OpCode::LdHeapArguments:
```

Comment:
Bytecode layout of this instruction is changed from Two Source Reg to One Source Reg.
Earlier it was taking property id array as one of the operands - Now we can retrieve the property id array of the formals from the FunctionBody directly.

---
In reply to: [68152909](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68152909) [](ancestors = 68152909)